prometheus-metrics/src/main/scala/org/http4s/metrics/prometheus/PrometheusExportService.scala

```diff
@@ -25,7 +25,7 @@ import java.io.StringWriter
 
 import org.http4s.Uri.Path
 import org.http4s._
-import org.http4s.implicits.http4sLiteralsSyntax
+import org.http4s.Http4s._
```

Comment:
Oh, wow, I forgot that exists.  That's from before Cats, and we might want to clean that up.  But, :+1: for what compiles now!